src/util/log.c

```diff
@@ -15,6 +16,34 @@
 #define unlikely(x)     (x)
 #endif
 
+static int caseInsensitiveStrncmp(char const * const string1, 
```

Comment:
The second const is without meaning.
Also, current convetion is to use `const char *string1`